spec/models/event_spec.rb

```diff
@@ -1,17 +1,121 @@
 require 'spec_helper'
 
 describe Event do
-  context '#users_available' do
-    context 'A event has users available' do
-      it 'returns a list with a user' do
-        stub_request(:any, 'https://api.twitter.com/1.1/statuses/update.json')
-        user = create(:user, interest_list: 'Ruby')
-        Availability.create(week_day: 5, start_time: DateTime.new(2015, 03, 06, 20, 00), end_time: DateTime.new(2015, 03, 06, 23, 30), user_id: user.id)
-        event = Event.create(name: 'Random Event', location: 'Braga', start_date: DateTime.new(2015, 03, 06, 20, 30), end_date: DateTime.new(2015, 03, 06, 21, 00), owner: user.id, interest_list: 'Ruby')
+  context '#attending?' do
+    context 'user has attendance to the event' do
+      it 'attends to the event' do
+        user = create :user
+        date_time_now = DateTime.now
+        one_hour_later = date_time_now + 1.hour
+        three_hours_later = date_time_now + 3.hour
 
-        result = event.notify_users
+        event = Event.create(name: 'Ruby-tapas', location: 'braga', start_date: date_time_now, end_date: one_hour_later, owner_id: user.id)
+        Availability.create(week_day: date_time_now.wday, start_time: date_time_now, end_time: three_hours_later, user_id: user.id)
+        Attendance.create(user_id: user.id, event_id: event.id)
 
-        expect(result).to eq([user])
+        user_attendding_to_event = event.attending?(user.id)
+
+        expect(user_attendding_to_event).to eq(true)
+      end
+    end
+
+    context "user hasn't attendance to the event" do
+      it "doesn't attend to the event" do
+        user = create :user
+        date_time_now = DateTime.now
+        one_hour_later = date_time_now + 1.hour
+        three_hours_later = date_time_now + 3.hour
+        event = Event.create(name: 'Ruby-tapas', location: 'braga', start_date: date_time_now, end_date: one_hour_later, owner_id: user.id)
+        Availability.create(week_day: date_time_now.wday, start_time: date_time_now, end_time: three_hours_later, user_id: user.id)
+
+        user_attendding_to_event = event.attending?(user.id)
+
+        expect(user_attendding_to_event).to eq(false)
+      end
+    end
+  end
+
+  context '#notify_users' do
+    context 'after create an event' do
+      context 'there are users with availability' do
```

Comment:
aren't there too many contexts here? What do you need the "after create an event" for?